### PR TITLE
Improving the work session form

### PR DESCRIPTION
**Pull request by bitamar**
#185

### CODE REVIEW

**Comment by bitamar** [comment]
screen shot 2015-12-14 at 15 17 51

File: productivity/modules/custom/productivity_timewatch/productivity_timewatch.module [hunk 1]
```diff
@@ -83,4 +88,74 @@ function productivity_timewatch_permission() {
...
+
+  // Display only active employees.
+  $uids = array();
+  $wrapper = entity_metadata_wrapper('node', $form['#node']);
+
+  // Get all active employees.
+  $query = new EntityFieldQuery();
+  $result = $query
+    ->entityCondition('entity_type', 'user')
+    ->propertyCondition('status', 1)
```
Comment (nader77) [inline productivity/modules/custom/productivity_timewatch/productivity_timewatch.module]: `1` => `NODE_PUBLISHED`

Comment (bitamar) [inline productivity/modules/custom/productivity_timewatch/productivity_timewatch.module]: It's a user, not a node

Comment (nader77) [inline productivity/modules/custom/productivity_timewatch/productivity_timewatch.module]: :+1: Ohh sorry.

**Comment by nader77** [comment]
Looking good, Merged.

**Pull request closed without merging**